README.md

```diff
@@ -1,22 +1,28 @@
 # Uber Rides Java SDK (Beta) [![Build Status](https://travis-ci.org/uber/rides-java-sdk.svg?branch=master)](https://travis-ci.org/uber/rides-java-sdk)
 This SDK helps your Java App make HTTP requests to the Uber Rides API.
 
+## Prerequisites
+- Dependency manager: Gradle or Maven
```

Comment:
It might be standard, but I saw many devs trying to use the SDK because there's no one available for the language of choice. I think we should also list the minimum JDK version for that matter. Would help sort out issues quickly in case they just don't meet the requirements ... Like, on SO, we could simply ask "did you check out the requirements?" and we might have fewer "real" issues to deal with.

I think listing the prerequisites doesn't hurt nor does is distract. We could even add a link to the dependency managers to get people to the docs, that might increase the usefulness of the list. Does that make sense?